src/Blockcore.Indexer.Cirrus/Client/Types/ReceiptResponse.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+
+namespace Blockcore.Indexer.Cirrus.Client.Types
+{
+    public class ReceiptResponse
+    {
+        public string TransactionHash { get; set; }
+      public string BlockHash { get; set; }
+      public ulong? BlockNumber { get; set; }
+      public string PostState { get; set; }
+      public ulong GasUsed { get; set; }
+      public string From { get; set; }
+      public string To { get; set; }
+      public string NewContractAddress { get; set; }
+      public bool Success { get; set; }
+      public string ReturnValue { get; set; }
+      public string Bloom { get; set; }
+      public string Error { get; set; }
+      public LogResponse[] Logs { get; set; }
+
+   }
+
+    public class LogResponse
```

Comment:
Why have all this classes in the same file, it just makes it harder to find

Reply:
this files where copied form cirrus fullnode so I left them this way